### PR TITLE
config: match current prod config by default

### PR DESCRIPTION
**Pull request by david-caro**
- [x] I have all the information that I need (if not, move to `RFC` and look for it).
- [x] I linked the related issue(s) in the corresponding commit logs.
- [x] I wrote [good commit log messages](https://github.com/torvalds/subsurface-for-dirk/blob/5f15ad5a86ada3c5e574041a5f9d85235322dabb/README#L92-L119).
- [x] My code follows the code style of this project.
- [x] I've added any new docs if API/utils methods were added.
- [x] I have updated the existing documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

Signed-off-by: David Caro <david@dcaro.es>

### CODE REVIEW

**Comment by david-caro** [comment]
ci test please

**Comment by david-caro** [comment]
ci test please

File: inspirehep/config.py [hunk 1]
```diff
@@ -52,10 +52,11 @@
 #   some ORCIDs -> "^(0000-0002-7638-5686|0000-0002-7638-5687)$"
 FEATURE_FLAG_ORCID_PUSH_WHITELIST_REGEX = '.*'
 FEATURE_FLAG_ENABLE_FUZZY_MATCHER = False
```
Comment (drjova) [inline inspirehep/config.py]: could you please also enable ``FEATURE_FLAG_ENABLE_FUZZY_MATCHER``?

Comment (david-caro) [reply inspirehep/config.py]: :+1: done

**Comment by david-caro** [comment]
I was thinking though that the fuzzy matcher will most probably not behave as expected on dev, as it depends on the actual contents of the ES instance. It will match too many things. I'll enable it and see if there are any issues in the current tests, but probably we might want to disable it if it misbehaves too much, as the actual results of it are not meaningful in dev (though the process is).

**Comment by david-caro** [comment]
ci test please

**Comment by david-caro** [comment]
ci test please

**Comment by david-caro** [comment]
ci merge please

**Pull request merged**